Models/CourseEnrollments.cs

```diff
@@ -13,14 +13,15 @@ public class CourseEnrollments
 
         [ForeignKey("person_system_identifier")]
         [Required]
-        public string student_id { get; set; }
-
-        [Required]
-        public DateTime course_start { get; set; } = DateTime.Now;
+        public string student_id { get; set; }   
 
         [ForeignKey("subject_id")]
         [Required]
-        public Subjects Subjects { get; set; }
```

Comment:
This is fine. You don't need to add the foreign key table annotation as EF Core already knows that it should be a foreign key as you are defining its type as `Subjects` and it would look for the primary key in that table. Also just a suggestion. Table annotations also work fine, just make sure you put the `[Foreign Key]` annotation at the right places or EF Core will struggle to identify what should be a foreign key and what should be primary key.